app/controllers/reviews_controller.rb

```diff
@@ -1,10 +1,15 @@
 class ReviewsController < ApplicationController
+
   def new
+    @review = Review.new
   end
 
   def create
+    @review = Review.new(review_params)
```

Comment:
need to define review_params in private method

Reply:
nice